stratum/hal/bin/np4intel/docker/build-stratum-np4intel-container.sh

```diff
@@ -56,16 +62,29 @@ fi
 
 BUILD_ARGS="--build-arg JOBS=$JOBS"
 
-# Grab the NP4 tarball
-echo """Copying NP4 tarball to $DOCKERFILE_DIR/
-NOTE: Copied tarballs will be DELETED after the build"""
+# Grab the NP4 binary
+echo """Copying NP4 binary to $DOCKERFILE_DIR/
+NOTE: Copied binary will be DELETED after the build"""
 
 if [ -n "$1" ]; then
-    NP4_TAR=$(basename $1)
-    BUILD_ARGS="$BUILD_ARGS --build-arg NP4_TAR=$NP4_TAR"
+    NP4_BIN=$(basename $1)
+    BUILD_ARGS="$BUILD_ARGS --build-arg NP4_BIN=$NP4_BIN"
     cp -f $1 $DOCKERFILE_DIR
 fi
 
+# Make sure the Netcope SDK binary matches our checksum
+if [ "$SKIP_MD5" != YES ]; then
+    NP4_MD5=${NP4_BIN:0:-4}.md5
+    echo "$NP4_CHKSUM $NP4_BIN" >$NP4_MD5
+    md5sum -c $NP4_MD5
```

Comment:
just no

Reply:
Well I guess that reads better, but will also require that we set NP4_MD5 to NO as the default if the switch is not set.